cypress/integration/individual-pages/yourBeaconRegistryAccount.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { PageURLs } from "../../../src/lib/urls";
+import { givenIDoNotHaveAValidSession } from "../common/selectors-and-assertions.spec";
+
+describe("Your beacon registry account", () => {
+  it("Redirects the user to the sign up or sign in page if the user does not have a valid session", () => {
```

Comment:
Wording on this should be slightly different as this doesn't redirect to the sign up or sign in page but the `unauthenticated` page 😄